src/CNLOADS.CPP

```diff
@@ -4499,9 +4469,9 @@ RC RSYS::rs_SetupASHP()		// set ASHP defaults and derived parameters
 	// inter-default cap95 / cap47
 	//   at least one is present (see rs_CkF)
 	if (!IsSet( RSYS_CAP95))
-		rs_cap95 = rs_Cap95FromCap47ASHP( rs_cap47);
+		ashpCap95FromCap47( rs_cap47, IsSet(RSYS_CAPRAT9547), rs_capRat9547);
```

Comment:
Should this not be rs_cap95 = ashpCap95FromCap47(  ...) ?

Reply:
Good catch. Don't know how I missed that. I think the scenario here is pretty unlikely (and part of why I didn't add something analogous to AIRHANDER, where currently both capacities are required).